go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.1
 	github.com/containers/buildah v1.22.1-0.20210824095355-fc1e256f5051
-	github.com/containers/common v0.44.0
+	github.com/containers/common v0.44.1-0.20210914173811-fcaa2e0de285
```

Comment:
If this is heade towards RHEL 8.5.0.2, we probably ought to cut a common v0.44.1

Reply:
Will cut a release for podman 3.4.